src/format.js

```diff
@@ -14,7 +14,7 @@ const weekdayFormat = memoize1((locale, weekday) => {
   return new Intl.DateTimeFormat(locale, {timeZone: "UTC", ...(weekday && {weekday})});
 });
 
-export function formatNumber(locale = "en-US") {
+function formatNumber(locale) {
```

Comment:
here I learned that we don't need to set the default locale (cf. #384) in this function, since it's only called from formatAuto.